<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>849</width>
    <height>684</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="search_label">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>12</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Search</string>
    </property>
   </widget>
   <widget class="QPushButton" name="search_btn">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>79</y>
      <width>31</width>
      <height>33</height>
     </rect>
    </property>
    <property name="text">
     <string>üîç</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="search_lineedit">
    <property name="geometry">
     <rect>
      <x>42</x>
      <y>80</y>
      <width>361</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="placeholderText">
     <string>Enter Text</string>
    </property>
   </widget>
   <widget class="QLabel" name="filter_label">
    <property name="geometry">
     <rect>
      <x>38</x>
      <y>130</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Filters</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="price_checkbox">
    <property name="geometry">
     <rect>
      <x>46</x>
      <y>449</y>
      <width>131</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Price Limit Range</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="incomecheckbox">
    <property name="geometry">
     <rect>
      <x>46</x>
      <y>505</y>
      <width>261</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Do you want to search in your incomes?</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="costcheckbox">
    <property name="geometry">
     <rect>
      <x>47</x>
      <y>572</y>
      <width>261</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Do you want to search in your costs?</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="price_high">
    <property name="geometry">
     <rect>
      <x>386</x>
      <y>446</y>
      <width>151</width>
      <height>27</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="placeholderText">
     <string>Enter High Price</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="price_low">
    <property name="geometry">
     <rect>
      <x>209</x>
      <y>446</y>
      <width>151</width>
      <height>27</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="placeholderText">
     <string>Enter Low Price</string>
    </property>
   </widget>
   <widget class="QPushButton" name="return_btn">
    <property name="geometry">
     <rect>
      <x>636</x>
      <y>564</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Return MainPage</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="custom_period_check">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>170</y>
      <width>111</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Custom Period</string>
    </property>
   </widget>
   <widget class="QCalendarWidget" name="calender_start">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>220</y>
      <width>344</width>
      <height>195</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="start_label">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>197</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QCalendarWidget" name="calender_end">
    <property name="geometry">
     <rect>
      <x>431</x>
      <y>220</y>
      <width>344</width>
      <height>195</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="end_label">
    <property name="geometry">
     <rect>
      <x>577</x>
      <y>196</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>End</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>849</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
